src/App.css

```diff
@@ -14,20 +14,151 @@
 }
```

Comment:
We must keep global styling at its minimum and put styles together with the component they belong